packages/app/src/contacts/components/contacts/contacts.component.tsx

```diff
@@ -56,6 +56,26 @@ import { ExportContactFailedModal } from "../export-contact-failed-modal/export-
 import { applyValidationRulesToImportedContacts } from "App/contacts/helpers/apply-validation-rules-to-imported-contacts/apply-validation-rules-to-imported-contacts"
 import { ExportContactsResult } from "App/contacts/constants"
 import DeleteContactsPopup from "./delete-contacts-popup/delete-contacts-popup.component"
+import _ from "lodash"
```

Comment:
```suggestion
import { differenceWith, isEqual } from "lodash"
```

- It seems like the import of a whole Lodash library will lead to the largest bundle size.

source: https://www.blazemeter.com/blog/import-lodash-libraries

Reply:
Sure, resolved in 875cf84fe